### ISSUE TITLE
Cannot read property 'toLowerCase' of null

### ISSUE THREAD

**Issue by websterguy** [Lavaeolous/PF1-StatBlock-Converter-Module#465]
Latest version of everything as of 6/15/21

```js
Uncaught (in promise) TypeError: Cannot read property 'toLowerCase' of null
    at Function.capitalize (sbcUtils.js:1127)
    at Function.generatePlaceholderEntity (sbcUtils.js:550)
    at spellBooksParser.parse (sbcParsers.js:2851)
    at async parseOffense (sbcParsers.js:2080)
    at async parseCategories (sbcParsers.js:161)
    at async Function.parseInput (sbcParser.js:212)
    at async Function.prepareInput (sbcParser.js:66)
    at async http:/localhost:31000/modules/pf1-statblock-converter/scripts/sbcInput.js:110
```
statblock:
```
Albor Voltiaro

CR 11

XP 12,800

Male human cleric of Dagon 12

CE Medium humanoid (human)

Init +4; Senses Perception +8

DEFENSE

AC 25, touch 14, flat-footed 25 (+9 armor, +4 deflection, +2 shield)

hp 105 (12d8+48)

Fort +11, Ref +7, Will +13

Immune mind-affecting effects; Resist cold 20, fire 30

OFFENSE

Spd 20 ft.

Melee trident of warning +16/+11 (1d8+6/19-20)

Ranged trident of warning +12 (1d8+6/19-20) or mwk silver dagger +10 (1d4+4/19-20)

Space 5 ft.; Reach 5 ft.

Special Attacks channel negative energy 4/day (DC 15, 6d6), fury of the Abyss* (+6, 7/day), scythe of evil (6 rounds, 2/day), surge* (+16, 7/day)

Cleric Spells Prepared (CL 12th; concentration +16)
6th—blade barrier (DC 20), cone of coldD (DC 20), harm (DC 20)
5th—ice stormD, slay living (2, DC 19), true seeing
4th—control waterD, divine power, freedom of movement, poison (DC 18), unholy blight (DC 18)
3rd—blindness/deafness (DC 17), dispel magic, magic vestment, rageD (DC 17), searing light, water breathing
2nd—bull's strength, death knell (DC 16), hold person (DC 16), resist energy, silence (DC 16), slipstream*, D
1st—bless, command (DC 15), doom (DC 15), obscuring mistD, protection from good, shield of faith
0 (at will)—create water, detect magic, guidance, resistance

D domain spell; Domains Evil (Demon subdomain*), Water (Oceans subdomain*)

TACTICS

Before Combat Albor casts magic vestment every day. Before combat, if he hears the sounds of battle elsewhere in the house, he casts resist energy (fire), bull's strength, and shield of faith, and activates his ring of force shield.

During Combat Albor casts divine power on the first round of combat. In battle, he attempts to soften foes up with channel energy and separate them with a blade barrier hidden within obscuring mist. He attempts to pick off foes one at a time from the mist with slay living and harm, using his area affect spells for opponents that remain grouped together.

Morale Albor fights until his hit points reach 0, at which point the slugspawn infesting his body matures into a spawning canker that erupts from his neck (see page 36).

STATISTICS

Str 18, Dex 10, Con 14, Int 12, Wis 19, Cha 8

Base Atk +9; CMB +13; CMD 27

Feats Combat Casting, Extra Channel, Improved Critical (trident), Improved Initiative, Lightning Reflexes, Toughness, Weapon Focus (trident)

Skills Diplomacy +8, Knowledge (history) +6, Knowledge (planes) +6, Knowledge (religion) +14, Perception +8, Sense Motive +13, Spellcraft +10, Swim +12

Languages Abyssal, Common

SQ slugspawn infested

Combat Gear wand of cure light wounds (13 charges); Other Gear masterwork chainmail; trident of warning; masterwork silver dagger; cloak of resistance +1; ring of force shield; goldthreaded chasuble (worth 75 gp); key to sea chest in area E7; fishgold medallion resembling a cross between an eel, a fish, and an octopus (key to the phase door in area F3); fishgold ceremonial tiara (worth 450 gp); fishgold unholy symbol of Dagon (worth 50 gp); 13 gp

SPECIAL ABILITIES

Slugspawn Infested (Ex) Albor has been infested with a slugspawn, granting him immunity to mind-affecting effects.

* See the Advanced Player's Guide.
```

**Comment by ehb-ahb** [comment]
Another statblock that fails in the same way.
------------------------------------------------
ANEVIA TIRABADE  CR 2 
XP  600 
Female human rogue 3 
NG Medium humanoid (human) 
Init +7; Senses Perception +5 
DEFENSE 
AC 14, touch 11, flat-footed 11 (+3 armor, -2 broken leg, +3 Dex) 
hp  17 (3d8) 
Fort +1, Ref +6, Will +O; +1 vs. traps 
Defensive Abilities evasion, trap sense + 1 
Weaknesses broken leg 
OFFENSE 
Speed 15 ft. 
Melee short sword +3 (1d6+1/19-20) 
Ranged shortbow +5 (1d6/x3) 
Special Attacks sneak attack +2d6 
Rogue Spell-Like Abilities (CL 3rd, concentration +5) 
3/day-/ight 
TACTICS 
During Combat Even before she broke her leg, Anevia preferred 
to fight with her bow. Although only four of her arrows 
survived the fall, Anevia relies on her shortbow for most fights 
until her leg is healed. Until she can secure at least 10 more 
arrows, she avoids using Rapid Shot, hoping to make every 
single shot count. If forced into melee, she does her best to 
flank with her sword, despite her limited mobility. 
Morale Unless she is fighting with a good friend or her wife 
Irabeth, Anevia flees to seek out a hiding spot to wait out 
her enemies if she falls below 5 hit points. 
STATISTICS 
Str 12, Dex  17, Con 10, Int 14, Wis 8, Cha 13 
Base Atk +2; CMB +3; CMD 16 
Feats Improved Initiative,  Point-Blank Shot, Rapid Shot 
Skills Acrobatics +5 (+1 when jumping), Climb +3, Diplomacy +7, 
Disable Device +9, Knowledge (dungeoneering) +8, 
Knowledge (local) +8, Knowledge (planes) +5, Linguistics +8, 
Perception +5, Sense Motive +5, Sleight of Hand +9, Stealth +9 
(+5 when moving), Swim -3 
Languages Common, Hallit, Orc, Shadowtongue 
SQ rogue talents (minor magic), trapfinding +1 
Combat Gear alchemist's fire (2), smokestick (3); Other Gear 
+1 leather armor, short sword, shortbow with 4 arrows, silk 
rope (50 ft.), thieves' tools, 46 gp

**Comment by Lavaeolous** [comment]
Fixed in 3.3.1.

The Statblock of "ANEVIA TIRABADE CR 2" is malformed, you may need to remove the extra linebreaks and change for example the Will Save, as this is using a capital letter "O" instead of the number "0"
